src/api/custom-resolvers/mutations/verify-account-reactivation-token/index.test.ts

```diff
@@ -54,17 +57,17 @@ describe('custom-resolvers/verify-account-reactivation-token', () => {
     const reactivationExpiry = ACCOUNT.REACTIVATION_EXPIRY();
 
     const unverifiedAndBlockedAccount = {
-      ...mockAccount,
-      [IS_VERIFIED]: false,
-      [IS_BLOCKED]: true,
+      ...mockAccountUpdate,
```

Comment:
`IS_VERIFIED` and `IS_BLOCKED` have been removed here and replaced with `mockAccountUpdate`, however the flags are different:

In the mocks:
```ts
isVerified: true,
isBlocked: false,
```

Which is the reverse of the original test.

Not sure if it should still pass or if the original flags are required (off the top of my head), can we check please?

Reply:
the status is removed at the top of the file and added on line 66 to the correct table.  line 66 has `[IS_VERIFIED]: false, [IS_BLOCKED]: true`